.github/workflows/ci.yml

```diff
@@ -0,0 +1,54 @@
+name: Node.js CI
+
+on:
+  pull_request:
+  schedule:
+    - cron: '0 0 * * 0'
```

Comment:
Why running on cron? I don't think we need that. What we need is to secure that each time we merge something to master the CI is ran. This is to secure the codecov page (see Readme.md).

Reply:
@kelson42 it's a weekly check in case any of our test develops a problem it could be noticed and solved(and not noticed when someone else  makes a PR).....but if you want I can remove it